qiskit/circuit/library/__init__.py

```diff
@@ -126,6 +126,43 @@
    ZGate
    GlobalPhaseGate
 
+
+get_standard_gate_name_mapping()
+-------------------
```

Comment:
docs are not building because [there is a warning](https://dev.azure.com/qiskit-ci/qiskit/_build/results?buildId=61157&view=logs):
```
/home/vsts/work/1/s/.tox/docs/lib/python3.9/site-packages/qiskit/circuit/library/__init__.py:docstring of qiskit.circuit.library:118: WARNING: Title underline too short.
```

```suggestion
get_standard_gate_name_mapping()
-------------------------------------
```

But if you move this documentation to `qiskit.circuit.library.standard_gates.get_standard_gate_name_mapping`, you probably dont have to worry about this specific error.

Reply:
Oh okay. I will do that.
Thank you